### PR TITLE
Update codeowners for JS / crypto

### PR DESCRIPTION
**Pull request by mitschabaude**
This updates the codeowners of some parts of the code:
* src/lib/snarky_js_bindings -> product-eng + @imeckler + @mrmr1993
* add product-eng to some other JS parts of the codebase
* src/lib/crypto -> crypto-eng

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -35,12 +35,15 @@
 /src/lib/mina_numbers/ @MinaProtocol/crypto-eng-reviewers
 /src/lib/crs/ @MinaProtocol/crypto-eng-reviewers
 /src/lib/crypto_params/ @MinaProtocol/crypto-eng-reviewers
+/src/lib/crypto @MinaProtocol/crypto-eng-reviewers
+/src/lib/crypto/kimchi_bindings/js @MinaProtocol/crypto-eng-reviewers @MinaProtocol/product-eng-reviewers
```
Comment (mrmr1993) [inline CODEOWNERS:39]: Should this be all of product eng, or just you @mitschabaude?

**Review by mrmr1993 — COMMENTED** [review]
A nit, but also: could you remove `@emberian` from the file to appease the Github linter?

File: CODEOWNERS (hunk 1 above)
Comment (mitschabaude) [reply CODEOWNERS:39]: Good question :) I can reduce it to me for now to make it more tight, and add other JS devs when they start contributing to those parts of the code base

**Review by bkase — APPROVED** [review]
Looks good to me

**Pull request merged**